### PR TITLE
More membership work

### PR DESCRIPTION
**Pull request by mayeaux**
Fix the way checks are made, with additional touchups

### CODE REVIEW

**Comment by tzarebczan** [comment]
This tested better, tracked much less check requests vs Odysee.com on similar app navigations..

File: ui/page/channels/view.jsx [hunk 1]
```diff
@@ -54,6 +55,12 @@ export default function ChannelsPage(props: Props) {
 
   return (
     <Page className="channelsPage-wrapper">
+      {/* TODO: use variabled spacing */}
+      <h1 className="section__title" style={{ marginBottom: '15px' }}>
+        Active channel
```
Comment (infinite-persistence) [inline ui/page/channels/view.jsx:60]: i18n

File: ui/page/odyseeMembership/view.jsx [hunk 2]
```diff
@@ -406,7 +418,7 @@ const OdyseeMembershipPage = (props: Props) => {
         <ul>
           <li>
             {__(
-              'Exclusive and early access features include: recommended content on homepage, livestreaming, and the ability to post odysee hyperlinks + images in comments. Account is also automatically eligible for Rewards. More to come later.'
+              'Exclusive and early access features include: recommended content on homepage, livestreaming, and the ability to post Odysee hyperlinks + images in comments. Account is also automatically eligible for Rewards. More to come later.'
```
Comment (infinite-persistence) [inline ui/page/odyseeMembership/view.jsx:421]: Update the corresponding/existing entry in app-string.json

**Review by infinite-persistence — COMMENTED** [review]
- [ ] noticed that it wasn't checking when expanding replies in comments
    - image

**Comment by tzarebczan** [comment]
see some of these scenarios to re-test too: https://github.com/OdyseeTeam/odysee-frontend/issues/1136#issuecomment-1074048418

**Pull request merged**